modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -83,6 +83,7 @@ public class JavaClientCodegen extends AbstractJavaCodegen
     public static final String OKHTTP_GSON = "okhttp-gson";
     public static final String RESTEASY = "resteasy";
     public static final String RESTTEMPLATE = "resttemplate";
+    public static final String RESTTEMPLATE_WITH_RETRIES = "resttemplate-with-retries";
```

Comment:
what about adding the feature to the resttemplate directly without creating another library option?